backends-velox/src/test/scala/io/glutenproject/execution/VeloxParquetWriteSuite.scala

```diff
@@ -37,19 +37,21 @@ class VeloxParquetWriteSuite extends WholeStageTransformerSuite {
           case _ => codec
         }
 
-        withTempPath { f =>
-          spark.table("lineitem").write
-            .format("velox")
-            .option("compression", codec)
-            .save(f.getCanonicalPath)
-          val files = f.list()
-          assert(files.nonEmpty, extension)
-          assert(files.exists(_.contains(extension)), extension)
+        TPCHTables.foreach { case (_, df) =>
```

Comment:
Can you help to delete the unit test in [TestOperator ](https://github.com/oap-project/gluten/blob/8f2c139700c3e253a65c4b2158a9a00809e1ddfd/backends-velox/src/test/scala/io/glutenproject/execution/TestOperator.scala#L95)here? It seems this test contains all the parquet write case. Thanks.

Reply:
sure, cleanup that